contracts/security-token/tests/InvestorInteractionContract.sol

```diff
@@ -69,7 +69,7 @@ contract InvestorInteractionContract is BogusAnnouncement, CheckpointToken, ERC8
     }
 
     if (options[_to] != 0) {
-      require(KYC.isWhitelisted(msg.sender));
+      require(KYC.getFlag(msg.sender, 0));
```

Comment:
What happens here?

Reply:
AFAIU, if a users has flag at bit position 0 means that a user has been whitelisted.
a flag a bit position one indicates that admin is allowed to transfer tokens to non-kyc'd users.

Reply:
Will add a comment: The KYC cleared flag is set